docs/layers.rst

```diff
@@ -28,7 +28,8 @@ first example. We start off by using `astropy.table
 comma-separated values (CSV) file of the data::
 
     >>> from astropy.table import Table
-    >>> OEC = 'https://worldwidetelescope.github.io/pywwt/data/open_exoplanet_catalogue.csv'
+    >>> OEC = 
+'https://worldwidetelescope.github.io/pywwt/data/open_exoplanet_catalogue.csv'
```

Comment:
I'm pretty sure this linewrap will break the doc formatting here; it should be undone.

Reply:
Changed.